apps/console/src/features/users/components/wizard/wizard-summary.tsx

```diff
@@ -129,7 +132,7 @@ export const AddUserWizardSummary: FunctionComponent<AddUserWizardSummaryProps>
                                 <Label.Group>
                                     {
                                         modifiedSummary.groups
-                                            .map((group, index) => (
+                                            .map((group:any, index:number) => (
```

Comment:
Do not use `any` type. Use proper type or make new interfaces

Reply:
Here the groups list is also of type any. This will be fixed after the release.